sdk/dotnet/Pulumi.Tests/Mocks/TestStack.cs

```diff
@@ -17,6 +18,23 @@ public sealed class InstanceArgs : Pulumi.ResourceArgs
     {
     }
 
+    public partial class MyCustom : Pulumi.CustomResource
```

Comment:
Why no `ResourceType` attribute here?

Reply:
The test isn't passing `MyCustom` as an input/output, so there isn't a need for the attribute on it.